Dashboard page

DenisShylin opened the pull request:
## Створені файли:
- src/Componts/Navigation.module.css
- src/Componts/Navigation.jsx
- hooks/useMedia.js
- public/dashboard_images/....
- також робота зі стилями в DashboardPage та Balance

- встановлена залежність: react-icons

DenisShylin committed:
style: fix styles in DashboardPage and ModalLogOut, add Header icons

DenisShylin committed:
Merge branch 'main' of github.com:DarkLordSith/MoneyG_Project_group-02 into DashboardPage

DenisShylin committed:
feat: add Navigation component and useMedia hook

DarkLordSith reviewed (changes requested):
- Перш за все, стягнути актуальний main
- Відкоригувати стилі відносно основних (кольори, шрифт, логотип і т.д)
- Прописати логіку відображення компонентів при натисканні на home, statistics та currency (на мобільній версії) * просто щоб не переводило на іншу сторінку а одразу відображало (якщо компонент за який ти не відповідаєш- відображай назву компонента при натисканні)
- Перенеси папку із зображеннями в src/assets/images/твоя папка (також видали зайвий логотип 
*в актуальній версії main його вже додано до public
- Спробуй завантажити іконки через бібліотеку react-icons (material design)

DenisShylin committed:
feat: create TransactionItem and TransactionList components with Redux integration

DenisShylin committed:
Merge branch 'main' of github.com:DarkLordSith/MoneyG_Project_group-02 into DashboardPage

DenisShylin committed:
fix: src/redux/finance/

DenisShylin committed:
fix: update styles and reorganize image assets

- Настроил стили в соответствии с основной веткой (цвета, шрифты, логотип и т. д.)
- Переместить папку с изображениями в src/assets/images/
- Удалил лишний логотип

DarkLordSith reviewed (changes requested):
Solid implementation on DashboardPage — great attention to detail!
Note: The branch is behind main. Please pull the latest changes from main and resolve merge conflicts to align with the current codebase.

DenisShylin committed:
Merge branch 'main' of github.com:DarkLordSith/MoneyG_Project_group-02 into DashboardPage

DenisShylin committed:
"pull: оновлення локальної гілки з main" (відалив коментар бонема шо пушити)
